packages/page-template-modal/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@automattic/page-template-modal",
+	"version": "1.0.0",
+	"description": "Automattic Page Template Modal",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"calypso:src": "src/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/page-template-modal"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"dependencies": {
```

Comment:
Can we use ranges instead of `*`? 

I know the versions used when this plugin loads in the browser depends on what Gutenberg provides in any case, but having version ranges at least makes explicit which versions we used to develop & test the plugin.

Reply:
ok, I got the versions from the ETK by running `yarn list` and applied them here